La pipeline ne peut être déployée

```
docker.pkg.github.com/mtes-mct/monitorfish/monitorfish-pipeline:v1.38.0
Traceback (most recent call last):
  File "/home/monitorfish-pipeline/datascience/main.py", line 4, in <module>
    from src.pipeline.flows_config import flows_to_register
  File "/home/monitorfish-pipeline/datascience/src/pipeline/flows_config.py", line 301, in <module>
    flow.storage = Local(
                   ^^^^^^
  File "/opt/pysetup/.venv/lib/python3.11/site-packages/prefect/storage/local.py", line 62, in __init__
    os.makedirs(abs_directory, exist_ok=True)
  File "<frozen os>", line 225, in makedirs
PermissionError: [Errno 13] Permission denied: '/home/monitorfish-pipeline/.prefect/flows'
make: *** [Makefile:87 : register-pipeline-flows-int] Erreur 1

```

_Originally posted by @louptheron in https://github.com/MTES-MCT/monitorfish/issues/2625#issuecomment-1798037248_

--- a/infra/docker/Dockerfile.DataPipeline
+++ b/infra/docker/Dockerfile.DataPipeline
@@ -95,6 +95,7 @@ RUN chmod +x /tini
 RUN useradd -m -r ${USER} && \
     chown ${USER} /home/${USER}
 WORKDIR /home/${USER}
+RUN mkdir /home/${USER}/.prefect/
 
 RUN chown -R ${USER} .
 USER ${USER}

